app/services/hyrax/workflow/mira_notification.rb

```diff
@@ -0,0 +1,27 @@
+module Hyrax
+  module Workflow
+    class MiraNotification < AbstractNotification
+      # regardless of what is passed in, set the recipients according to this notification's requirements
+      def initialize(entity, comment, user, recipients)
+        super
+        @recipients = workflow_recipients
+      end
+
+      def workflow_recipients
+        raise NotImplementedError, "Implement workflow_recipients in a child class"
+      end
+
+      # The Users who have an admin role
+      # @return [<Array>::User] an Array of Hyrax::User objects
```

Comment:
It think the correct Yard collection annotation syntax for this is `[Array<User>]`. I'm not too worried about getting a fix in for merge.